website/docs/reference/resource-configs/trino-configs.md

```diff
@@ -3,11 +3,17 @@ title: "Starburst/Trino configurations"
 id: "trino-configs"
 ---
 
+## Cluster requirements
+
+The designated cluster must have an attached catalog where ojects such as tables and views can be created, renamed, altered, and dropped. And, any user connecting to the cluster with dbt must also have these same permissions for the target catalog.
+
 ## Session properties
 
-In the Starburst/Trino environment, you can set session properties that temporarily modifies the current session you have with the cluster. Refer to [Set Session](https://docs.starburst.io/latest/sql/set-session.html) in the Starburst/Trino docs for more details.
+In the Starburst/Trino environment, you can set session properties that modify the current session you have with the cluster. Refer to [Set Session](https://docs.starburst.io/latest/sql/set-session.html) in the Starburst/Trino docs for more details.
+
+The standard way to define session properties is via the `session_properties` field of your `profiles.yml`. This ensures that all dbt connections use these settings by default.
```

Comment:
is with the

Reply:
Thats also a problem btw.. it affects all connections .. and therefore potentially different cluster ... not sure if it makes sense to use this as a general recommendation. wdyt @hovaesco ?

Reply:
The general recommendation should be to setup as much as possible on the server side. Session properties would be only required in the specific use cases (not common I'd say).